assets/platforms/fortinet.yaml

```diff
@@ -0,0 +1,40 @@
+---
+platform-type: 'fortinet'
```

Comment:
this should probably be fortinet_fortios (or im assuming this is fortios not the wlc or something else) for consistency with scrapli community/general pattern.

Reply:
sure, name can be changed, I didnot use scrapli before, just refer netmiko.

Reply:
I have remove fortinet in `definition.go` for now.let me do more tests on this, once I confirm with configuration mode, I will make a another PR for fortigate firewall. thanks